tests/soak-tests/src/test/java/org/apache/activemq/artemis/tests/soak/brokerConnection/mirror/ReplicatedBothNodesMirrorTest.java

```diff
@@ -185,20 +188,25 @@ private static void createMirroredServer(String serverName,
       brokerProperties.put("AMQPConnections." + connectionName + ".connectionElements.mirror.sync", "false");
       brokerProperties.put("largeMessageSync", "false");
 
-      brokerProperties.put("addressSettings.#.maxSizeMessages", "50");
-      brokerProperties.put("addressSettings.#.maxReadPageMessages", "2000");
-      brokerProperties.put("addressSettings.#.maxReadPageBytes", "-1");
-      brokerProperties.put("addressSettings.#.prefetchPageMessages", "500");
+      brokerProperties.put("mirrorAckManagerQueueAttempts", "5");
+      brokerProperties.put("mirrorAckManagerPageAttempts", "500000");
```

Comment:
Could this be set in the later 'if (paging)' block ?

Reply:
actually nope... the pageAttempt will also perform a regular queue attempt.

I wouldc set mirrorAckQueueAttempts to 500000 if not paging.. I will just keep it as this for both cases

Reply:
Given there is already a boolean indicating the paging/not scenario, I'd personally go with specific config for the test scenario. It seems unexpected as-is, I dont even really understand what it is actually going to do.

If not case-specific config, then perhaps an explaining comment why it is the way it is and what it would do would help, both now and perhaps more importantly later. Setting this paging related setting which defaults to 2 to 500000 instead when not paging, is not immediately obvious.